### DISCUSSION TITLE
Initializing a Codespace

### DISCUSSION THREAD

**Discussion by bartr** [community/community#15519] [Codespaces]
We need to run some commands when the Codespace boots. One of the commands is `mount -a`.

What is the best way to do that?

I've tried creating a script in /etc/init.d but it doesn't get executed. So I tried modifying an existing script and they don't appear to get executed either. They are copied to the /etc/rcX.d directories and if you run with sudo service, they work.

I know I can run in post-start but that's not early enough for some of our commands.

**Comment by ozyman42** [comment]
https://docs.github.com/en/codespaces/customizing-your-codespace/personalizing-github-codespaces-for-your-account#dotfiles
